adhocracy4/comments_async/static/modals/UrlModal.jsx

```diff
@@ -3,16 +3,35 @@ import django from 'django'
 
 import Modal from '../../../static/Modal'
 
+const translated = {
+  share: django.gettext('Share')
+}
+
 export const UrlModal = (props) => {
   const [clicked, setClicked] = useState(false)
   const buttonTextCopy = django.gettext('Copy')
   const buttonTextCopied = django.gettext('Copied')
+
+  const copyUrl = () => {
+    const copyText = document.getElementById('share-url-'.concat(props.objectId))
+    if (copyText) {
+      navigator.clipboard.writeText(copyText.value)
+        .then(() => {
+          setClicked(true)
+          return true
+        })
+        .catch((err) => {
+          console.error('Failed to copy text: ', err)
+        })
+    }
+  }
+
   const partials = {}
   partials.hideHeader = true
   partials.hideFooter = true
   partials.title = props.title
   partials.body = (
-    <div className="input-group">
+    <div className="input-group a4-modal__url-modal">
```

Comment:
Maybe call it `url-body` instead? Technically the modal is the very top element, right? Or give the modal a `className` prop and pass it there?